plugins/dai-release-backend/src/api/ReleaseOverviewApi.ts

```diff
@@ -257,4 +262,68 @@ export class ReleaseOverviewApi {
   async getReleaseInstances() {
     return this.config.getInstanceList();
   }
+
```

Comment:
Can be added in separate file. TemplatesApi.ts and move FolderApis to FolderApi.ts file and its test cases. Created a story for that along with refresh button.

https://www7.v1host.com/V1Production/assetdetail.v1?number=S-113627